src/main/java/appeng/items/parts/PartType.java

```diff
@@ -62,7 +62,7 @@
 import appeng.parts.p2p.PartP2PIC2Power;
 import appeng.parts.p2p.PartP2PItems;
 import appeng.parts.p2p.PartP2PLight;
-import appeng.parts.p2p.PartP2PLiquids;
+import appeng.parts.p2p.PartP2PFluids;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Wrong order for 'appeng.parts.p2p.PartP2PFluids' import. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)